mongo/bounties/bboard_v4.json

```diff
@@ -0,0 +1,351 @@
+[
```

Comment:
Seeding script, includes customerId field and reward.amountWithoutScale required for the BB to function with multitenancy.